src/System.Data.SqlClient/tests/FunctionalTests/project.json

```diff
@@ -2,7 +2,10 @@
   "dependencies": {
     "Microsoft.CSharp": "4.0.1-rc3-24201-01",
     "Microsoft.NETCore.Platforms": "1.0.1-rc3-24201-01",
+    "runtime.native.System.Data.SqlClient.sni": "4.0.0-rc3-24201-01",
```

Comment:
This might start getting out of sync given our new package split between TFS and open so we may need to remove this dependency, it shouldn't be directly needed any longer. I'm actually removing it in master right now.